_includes/selector.html

```diff
@@ -427,7 +427,8 @@
                 var dask_prefix = this.active_release === "Nightly" ? "dask/label/dev::" : "";
                 var pkg_names = {
                     "Plotly Dash": "dash",
-                    "Dask-SQL": dask_prefix + "dask-sql"
+                    "Dask-SQL": dask_prefix + "dask-sql",
+                    "PyTorch": "pytorch=*=*cuda*"
```

Comment:
We probably need single quotes to avoid shell expansion.
```suggestion
                    "PyTorch": "'pytorch=*=*cuda*'"
```